graylog2-web-interface/src/components/users/TokenList.tsx

```diff
@@ -16,6 +16,7 @@
  */
 import React, { useMemo, useState } from 'react';
 import styled from 'styled-components';
+import type User from 'src/logic/users/User';
```

Comment:
We usually import from `logic/users/User` with no src I am not sure if that causes any problem